packages/flutter_tools/lib/src/version.dart

```diff
@@ -254,6 +257,8 @@ class FlutterVersion {
     try {
       final String branch = _channel == 'alpha' ? 'alpha' : 'master';
       final DateTime remoteFrameworkCommitDate = DateTime.parse(await FlutterVersion.fetchRemoteFrameworkCommitDate(branch));
+      // fire and forget since nothing subsequent depends on it. Hope there's a flush/sync mechanism.
+      // ignore: unawaited_futures
```

Comment:
It looks ok in this case but as a general rule prefer to put the //ignore at the end of the offending line rather than before it